src/seedu/addressbook/AddressBook.java

```diff
@@ -712,14 +713,18 @@ private static void createFileIfMissing(String filePath) {
 
         showToUser(String.format(MESSAGE_ERROR_MISSING_STORAGE_FILE, filePath));
 
-        try {
+        createStorageFile(filePath, storageFile);
+    }
+
+	private static void createStorageFile(String filePath, final File storageFile) {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.